fonts/Makefile.am

```diff
@@ -39,10 +39,10 @@ gresym: gresym.ttf
 
 greextra: greextra.ttf
 
-clean:
-	@$(TOP_LEVEL_MAKE) $(RM) -f $(TTFFILES)
+clean-local:
+	$(RM) -f $(TTFFILES)
 
-install:
+install-data-local:
```

Comment:
The purpose for having this here was for people used to the old method of going into the fonts directory and typing `make install`.  The system would then tell them about the new method of installing.  By changing this target this way, it defeats the purpose of having an install target in the first place.